### ISSUE TITLE
refactoring: moving versions into properties section

### ISSUE THREAD

**Issue by Marfir** [buchen/portfolio#3244]
**current**
pom.xml of portfolio-app contains a lot of plugins in the pluginManagement section, but only one plugin has a version placeholder.
This makes it harder to find out if there is something outdated. e.g. there is an update for jacoco (0.8.7 -> 0.8.8)

**goal**
put the versions with a placeholder into the properties section

**Comment by buchen** [comment]
Good point. As Maven only contains the tooling dependencies (all others are defined in the target platform), I did not bother to extract them into properties yet. Every once in a while I run the "display-plugin-updates" and change the values. But version of course leak into the sub projects. I am happy to merge a pull request on this.
